### PR TITLE
[Modular] Revert borg language changes

### PR DESCRIPTION
**Pull request by Stalkeros2**
## About The Pull Request
Good morning pepol; woke up and saw this.
Reverts #22169's changes to modular files.

## How This Contributes To The Skyrat Roleplay Experience
The PR's changes are entirely opposed to Skyrat's design of silicons and the crew, and is completely detrimental to player interactions. I'd write more but I didn't have my breakfast so can't write a lot and why would you need a lot of text the first sentence is effectively the tl;dr of the whole reasoning.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image

</details>

## Changelog

:cl: Stalkeros
add: Borg languages are back.
/:cl:

### CODE REVIEW

**Comment by Iamgoofball** [comment]
It hasn't even hit live yet, why don't we see how it goes first?

**Comment by Stalkeros2** [comment]
> It hasn't even hit live yet, why don't we see how it goes first?

I'm fairly certain that there's no need for "seeing how it goes" when that was a solution for the upstream's issue and not the downstream's.

**Comment by Iamgoofball** [comment]
it's an issue down here as well since we restrict the number of languages you can have active at once here too, cyborgs and encryption keys let people bypass that limit

everyone knows common and cyborgs know common(and uncommon for people w/ Foreigner as human) so everyone can communicate with cyborgs already

**Comment by HoodMilk** [comment]
There's usually 2-3 borgs a round, on top of an AI (except for lowpop) and they're the easiest option for translation by far. Taking away this ability lets some of the lesser used ways to translate shine, like curators or PAI. I think the language restriction creates more interesting RP scenarios rather than taking them away. Even if there's some people cyborgs can't understand (The only situation of this now would be people intentionally speaking in an alternate language or ashwalkers), you can still communicate using emotes anyway.

**Comment by chemistrymain2** [comment]
> There's usually 2-3 borgs a round, on top of an AI (except for lowpop) and they're the easiest option for translation by far. Taking away this ability lets some of the lesser used ways to translate shine, like curators or PAI. I think the language restriction creates more interesting RP scenarios rather than taking them away. Even if there's some people cyborgs can't understand (The only situation of this now would be people intentionally speaking in an alternate language or ashwalkers), you can still communicate using emotes anyway.

No, it's just unfun. Sometimes I enjoy having private conversations with other species in their preferred language, and it creates RP that way.

Viewing languages this way like 'the ways to translate' is for ashwalkers or icewalkers. Most of the time, languages aren't used for this, because that sort of mechanical focus is more TG's thing. Instead, languages are used for RP.

The curator and pAi can speak in ashwalker and icecat and pirate and all the other important antag languages by default-there's no reason to lock borgs out of being able to speak chitinclick or draconic if they want to talk to a mothman or a lizardperson.

It's actually creating the opposite of interesting RP scenarios. It's just boring and unfun, speaking as a borg.

The people we already couldn't speak to-ashwalkers, icecats-the curator served the purpose of translation, there, as did pAis. 

For us, though, we're a _roleplay_ server, and being able to speak other languages to talk to a species in their preferred language has been part of the fun.

> people intentionally speaking in an alternate language

Yes, some people do that-for privacy's sake, or because they RP they are more fluent in those languages than that.

Can you imagine calling the curator to a private emotional conversation to translate for you when some guy is saying 'i'd be able to communicate this better in my native tongue'? No, that's a TG-style thing. That's purely mechanical.

**Comment by chemistrymain2** [comment]
> It hasn't even hit live yet, why don't we see how it goes first?

It's hit live.

It sucks.

**Comment by HoodMilk** [comment]
Fair enough I don't play cyborgs, but borgs should probably only know languages you can actually pick roundstart currently. After the language change happened they still know some of the old languages that crew can't even pick like shadowtongue, mushroom, or nekomimetic (Tbh not sure whey they're even in the code anymore they aren't used).

**Comment by Iamgoofball** [comment]
>The curator and pAi can speak in ashwalker and icecat and pirate and all the other important antag languages by default-there's no reason to lock borgs out of being able to speak chitinclick or draconic if they want to talk to a mothman or a lizardperson.

We're actually removing that from pAI upstream already, pAIs are going to inherit their current holder's languages.
Curators get to know every language because they're the librarian, aka massive nerd. They're also a one slot role and potentially an antagonist.

**Comment by SarmentiCampbell** [comment]
It is horrible, hope this get merged pretty fast

**Comment by PhonyName** [comment]
This needs to get merged ASAP, this reeks of "Antag run ruined, time to remove" salt PR. I fail to see the rationale for this PR being added, and it should be reverted.

Wanna have a private conversation with a silicon in the room? Law 2, fuck off borg.
Wanna use the curator? Ask the curator.
All this does is diminish the already limited silicon RP capacity.

Pls retire again.

File: modular_skyrat/master_files/code/modules/language/language_holder.dm [hunk 1]
```diff
@@ -54,3 +54,62 @@ GLOBAL_DATUM_INIT(language_holder_adjustor, /datum/language_holder_adjustor, new
...
+							/datum/language/beachbum = list(LANGUAGE_ATOM),
+							/datum/language/mushroom = list(LANGUAGE_ATOM),
+							/datum/language/shadowtongue = list(LANGUAGE_ATOM),
+							/datum/language/buzzwords = list(LANGUAGE_ATOM),
+							/datum/language/terrum = list(LANGUAGE_ATOM),
+							/datum/language/sylvan = list(LANGUAGE_ATOM),
+							/datum/language/siiktajr = list(LANGUAGE_ATOM),
+							/datum/language/canilunzt = list(LANGUAGE_ATOM),
+							/datum/language/akulan = list(LANGUAGE_ATOM)
+							)
```
Comment (GoldenAlpharex) [inline modular_skyrat/master_files/code/modules/language/language_holder.dm]: Please format these lists properly, so they aren't so much of an eyesore, and incredibly indented for no reason.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Comment by SarmentiCampbell** [comment]
On another note, You also forgot to do a removal on the original language holder

**Comment by GoldenAlpharex** [comment]
> On another note, You also forgot to do a removal on the original language holder

That isn't necessary, this is an override.

File: modular_skyrat/master_files/code/modules/language/language_holder.dm (hunk 1 above)
Comment (GoldenAlpharex) [reply modular_skyrat/master_files/code/modules/language/language_holder.dm]: Fixed it myself.

**Review by GoldenAlpharex — APPROVED** [review]
Should be good to go now.

**Comment by GoldenAlpharex** [comment]
For reference, merging this because it's an override we had in the code on our side, which means it was intentional, so it has to be removed intentionally in its own PR.

**Pull request merged**